### PR TITLE
Updated quests and map description.

### PR DESCRIPTION
**Pull request by Jaccouille**
$changelog: Added quest log to credits external help.
$changelog: Updated map description to include information on tribe size.

Many pub come into lobby and say "I thought this map used to be played with 3 team", so I updated map description to include "You can set the number of player per team using ingame mode with the -ppt 2, 3, 4, 5 or 6 command." hoping that people will read and understand that you can set the team in game.

### CODE REVIEW

File: wurst/systems/modes/Quests.wurst [hunk 1]
```diff
@@ -5,44 +5,73 @@ import LinkedList
...
+    "HiveWorkShop forum & discord:".color(HIGHLIGHT_COLOR)  + " https://www.hiveworkshop.com/".color(ENERGY_COLOR),
+    "Assets credits:".color(HIGHLIGHT_COLOR),
+    "General Frank " .color(HIGHLIGHT_COLOR) + "for Spirit of water/wind/darkness and gem of knowledge",
+    "Sin'dorei300 "  .color(HIGHLIGHT_COLOR) + "for iron/steel armor, skin coat and sniff icon",
+    "Matarael "      .color(HIGHLIGHT_COLOR) + "for iron/steel/anabolic boots and glove model",
+    "Blood raven "   .color(HIGHLIGHT_COLOR) + "for bone coat",
+    "Usedwell "      .color(HIGHLIGHT_COLOR) + "for hydra coat",
+    "Tranquil "      .color(HIGHLIGHT_COLOR) + "for bear trap",
+    "olofmoleman "   .color(HIGHLIGHT_COLOR) + "for disco duck model",
+    "RightField "    .color(HIGHLIGHT_COLOR) + "for cooked meat and snake mode",
```
Comment (HallerJohn) [inline wurst/systems/modes/Quests.wurst]: I'm assuming this is a typo and meant to say model

Comment (Jaccouille) [reply wurst/systems/modes/Quests.wurst]: Indeed

**Comment by jonathanmclaus** [comment]
These changes seem a bit too precise. I would just update suggested players to say "Configurable in-game." or something short and simple. Mentioning modes seems a bit technical and people might miss that. You could also update the force names in the lobby to reflect this.

The quest for credits seems to have a lot of hard coded stuff. I'd rather see a simple configuration data structure, such a mapping from string name to a list of string assets and generate the quest based on that, similar to changelog.

**Comment by Jaccouille** [comment]
I agree for the quest credits suggestion, but I don't think it's a priority at this moment, I did it so I could update the Hiveworkshop thread later on.

File: wurst.build [hunk 2]
```diff
@@ -10,9 +10,9 @@ buildMapData:
   scenarioData:
     description: Island Troll Tribes is a PvP-centric survival map in which multiple
       tribes of trolls compete to gather resources, craft equipment, and eradicate
-      each other. Join our Discord server at https://discord.gg/Rh9JdKs for mustering lobbies, reporting bugs, and making
-      suggestions.
-    suggestedPlayers: 6v6
+      each other. You can set the number of player per team using ingame mode with the -ppt 2, 3, 4, 5 or 6 command.
```
Comment (jonathanmclaus) [inline wurst.build]: Say `mode` insted of `command` to make it clearer.

**Pull request merged**